packages/extension-koni-ui/src/util/dom.ts

```diff
@@ -0,0 +1,20 @@
+// [object Object]
```

Comment:
![image](https://user-images.githubusercontent.com/92499846/161699000-51a58053-3c7d-4b04-9042-4dcfcbc8780e.png)
Incorrect header. Please run "yarn lint" for more details